.golangci.yml

```diff
@@ -119,6 +119,13 @@ linters-settings:
           files:
             - "!**/*_test.go"
 
+  gosec:
+    excludes:
+      # https://github.com/golangci/golangci-lint/issues/4735
+      - G601
+      # https://github.com/golangci/golangci-lint/issues/4735
+      - G113
```

Comment:
Please create an issue on our side to remove this excludes later.

Reply:
Updated comment with issue in this repo